CODEOWNERS

```diff
@@ -36,4 +36,4 @@
 # For more details, read the following articles on GitHub:
 # https://help.github.com/articles/about-codeowners/
 # https://github.com/blog/2392-introducing-code-owners/
-* @khlifi411 @jeremyharisch @ruanxin @tobiscr @clebs @Tomasz-Smelcerz-SAP @jakobmoellersap @adityabhatia @janmedrek
\ No newline at end of file
+* @khlifi411 @jeremyharisch @ruanxin @Tomasz-Smelcerz-SAP @lindnerby @janmedrek
```

Comment:
We should add `@kyma-project/jellyfish` as codeower, then we don't have to deal with this case later again.

Reply:
check lifecycle-manager https://github.com/kyma-project/lifecycle-manager/blob/main/CODEOWNERS

Reply:
Sure, done.